example/magefile.go

```diff
@@ -0,0 +1,50 @@
+// +build mage
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/magefile/mage/mg" // mg contains helpful utility functions, like Deps
+	"os"
+	"os/exec"
+	"time"
+)
+
+// Default target to run when none is specified
+// If not set, running mage will list available targets
+// var Default = Build
+
+func Timeout(ctx context.Context) {
+	fmt.Printf("timeout start\n")
+	time.Sleep(1000000000000)
```

Comment:
time.sleep takes a time.Duration, you could make this easier to read by just doing time.Sleep(time.Hour) or something :)

Reply:
It does and "oops!" did not intend to commit that... I'll remove it. I use that for my manual test cases.